developer-docs-site/docs/tutorials/full-node/run-a-fullnode.md

```diff
@@ -97,7 +97,8 @@ With your development environment ready, now you can start to setup your FullNod
 
 5. Checkout the `devnet` branch using `git checkout --track origin/devnet`.
 
-6. Make sure your current working directory is `aptos-core`. Copy the YAML configuration file from `config/src/config/test_data/public_full_node.yaml` to your current working directory. You will edit this file to ensure that your FullNode:
+6. Make sure your current working directory is `aptos-core`. 
+   Run `cp config/src/config/test_data/public_full_node.yaml fullnode.yaml` to create a copy of the fullnode config template. You will edit this file to ensure that your FullNode:
```

Comment:
Could you please also update the upgrade instruction for this change? https://aptos.dev/tutorials/full-node/update-fullnode-with-new-releases

Reply:
done